rust/blockstore/src/arrow/block/types.rs

```diff
@@ -36,6 +37,12 @@ pub struct Block {
     pub id: Uuid,
 }
 
+impl Value for Block {
```

Comment:
I think Value here is a bit confusing since the blockstore also has an internal Value trait